docs/src/assets/scss/foundations.scss

```diff
@@ -10,6 +10,7 @@ h4:target,
 h5:target,
 h6:target {
     background-color: var(--lighter-background-color);
+    border-radius: var(--border-radius);
```

Comment:
Could we also add some padding, like `padding: 3px 6px`?

Reply:
Sure, I'll check out various styles and leave comments!

Reply:
@amareshsm I've updated it.